packages/docs/docs/guides/rules-of-jsx.md

```diff
@@ -60,7 +60,7 @@ function* OrgData() {
 
 ## Component API
 
-Components take props as the first argument and ComponentContext (`packages/ai-jsx/src/index.ts`) as the second:
+Components take props as the first argument and ComponentContext ([`packages/ai-jsx/src/index.ts`](https://github.com/fixie-ai/ai-jsx/blob/main/packages/ai-jsx/src/index.ts)) as the second:
```

Comment:
Thanks. This is better. But now that we have API docs, the best thing is actually to kill the link to `index.ts` and instead link to the API docs entry for `ComponentContext`. (And do the same thing everywhere else we reference types from the API. :smile:)

Reply:
Right, that makes sense. It kinda forces me to look at the API docs and see if I need to improve them as well.